app/helpers/alerts_helper.rb

```diff
@@ -1,3 +1,11 @@
 # frozen_string_literal: true
 module AlertsHelper
```

Comment:
Since this is a helper for the AlertsListController's index view, we should probably rename this class AlertsListHelper (and change the file name). With that change, there will also be no possibility of confusion over whether the include the module in the controller.